src/plots/gl3d/scene.js

```diff
@@ -388,7 +392,7 @@ proto.render = function() {
             }
```

Comment:
Right above here `toPrecision(3)` is used for formatting `norm` and `divergence` instead of `Axes.hoverLabelText`. Should we keep/change that?

Reply:
In principle yes but that's extremely low priority IMO